unit-tests/func/hw-errors/test-hw-errors-sanity.cpp

```diff
@@ -0,0 +1,17 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
I think "hw-errors" in the filename is redundant: the the final test name will be `test-func-hw-errors-hw-errors-sanity.exe`

Reply:
done